_includes/footer.html

```diff
@@ -3,17 +3,35 @@
     <div class="row">
       <div class="col-md-3 text-center">
         <ul class="list-inline">
-          <li><a target="_blank" href="https://www.facebook.com/PyConIndia/">{% asset fb.png alt="Facebook" %}</a></li>
-          <li><a target="_blank" href="https://twitter.com/pyconindia">{% asset tw.png alt="Twitter" %}</a></li>
+          <li><a target="_blank" href="https://www.facebook.com/PyConIndia/">
+            <picture>
+              {% srcset fb.webp %}
+              {% asset fb.png alt="Facebook" %}
+            </picture>
+          </a></li>
+          <li><a target="_blank" href="https://twitter.com/pyconindia">
+            <picture>
+              {% srcset tw.webp %}
+              {% asset tw.png alt="Twitter" %}
+            </picture>
+          </a></li>
         </ul>
         <a target="_blank" href="mailto:contact@in.pycon.org">contact@in.pycon.org</a>
       </div>
       <div class="col-md-6 text-center">
-        {% asset logo-small.png alt="PyCon India 2019 Logo" width="70%" %}
+        <picture>
```

Comment:
I am not very comfortable with this either. 

Ideally, we should have a custom tag(say `image` instead of `srcset`) and it should be invoked like

`{% image src=logo-small.png alt="PyCon India Logo" %}`

and that should render

```
<picture>
    <source srcset="./assets/images/logo-small.png" type="image/png>
    <source srcset="./assets/images/logo-small.webp" type="image/webp">
</picture>
```

Like we discussed earlier, if generating webp images is the issue, we can offload it from this PR. Let's manually check-in webp equivalents with their checksum. I don't really think we are going to keep adding images from now. That way, the only thing that needs to be done is implementing this custom tag. I can help with manually generating webp images if you want. 

Let me know what you think.

Reply:
Manually generating webp isn't quite the problem I am facing. I've update PR, need a little help with the checksum part though